Cannot compile

`cmake` output:
```
-- The C compiler identification is GNU 9.1.1
-- The CXX compiler identification is GNU 9.1.1
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Boost version: 1.69.0
-- Found the following Boost libraries:
--   filesystem
--   program_options
--   regex
--   system
-- Found GDAL: /usr/lib64/libgdal.so (found suitable version "2.3.2", minimum required is "2.0") 
-- GDAL found and working properly.
-- Configuring done
-- Generating done
-- Build files have been written to: /home/frafra/Downloads/ijgis_ca_carving/build
```


`make` output:
```
Scanning dependencies of target test_gdal
[  1%] Building CXX object cmake_extras/CMakeFiles/test_gdal.dir/test_gdal.cpp.o
[  2%] Linking CXX executable ../bin/test_gdal
[  2%] Built target test_gdal
Scanning dependencies of target run_test_gdal
[  4%] Generating test_gdal.log
[  4%] Built target run_test_gdal
Scanning dependencies of target coordinates
[  5%] Building CXX object src/framework/CMakeFiles/coordinates.dir/coordinates.cpp.o
[  7%] Linking CXX static library libcoordinates.a
[  7%] Built target coordinates
Scanning dependencies of target ReferenceSystem
[  8%] Building CXX object src/framework/CMakeFiles/ReferenceSystem.dir/ReferenceSystem.cpp.o
[ 10%] Linking CXX static library libReferenceSystem.a
[ 10%] Built target ReferenceSystem
Scanning dependencies of target directions
[ 11%] Building CXX object src/framework/CMakeFiles/directions.dir/directions.cpp.o
[ 13%] Linking CXX static library libdirections.a
[ 13%] Built target directions
Scanning dependencies of target global_parameters
[ 14%] Building CXX object src/framework/CMakeFiles/global_parameters.dir/global_parameters.cpp.o
[ 15%] Linking CXX static library libglobal_parameters.a
[ 15%] Built target global_parameters
Scanning dependencies of target geo
[ 17%] Building CXX object src/framework/CMakeFiles/geo.dir/geo.cpp.o
[ 18%] Linking CXX static library libgeo.a
[ 18%] Built target geo
Scanning dependencies of target Area
[ 20%] Building CXX object src/framework/CMakeFiles/Area.dir/Area.cpp.o
/home/frafra/Downloads/ijgis_ca_carving/src/framework/Area.cpp: In member function ‘virtual double geo::Area::min_accepted_overlap() const’:
/home/frafra/Downloads/ijgis_ca_carving/src/framework/Area.cpp:92:21: error: ‘numeric_limits’ is not a member of ‘std’
   92 |         return std::numeric_limits<double>::min();
      |                     ^~~~~~~~~~~~~~
/home/frafra/Downloads/ijgis_ca_carving/src/framework/Area.cpp:92:36: error: expected primary-expression before ‘double’
   92 |         return std::numeric_limits<double>::min();
      |                                    ^~~~~~
/home/frafra/Downloads/ijgis_ca_carving/src/framework/Area.cpp:92:36: error: expected ‘;’ before ‘double’
   92 |         return std::numeric_limits<double>::min();
      |                                    ^~~~~~
      |                                    ;
/home/frafra/Downloads/ijgis_ca_carving/src/framework/Area.cpp:92:42: error: expected unqualified-id before ‘>’ token
   92 |         return std::numeric_limits<double>::min();
      |                                          ^
make[2]: *** [src/framework/CMakeFiles/Area.dir/build.make:63: src/framework/CMakeFiles/Area.dir/Area.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:429: src/framework/CMakeFiles/Area.dir/all] Error 2
make: *** [Makefile:95: all] Error 2
```

--- a/src/framework/Area.cpp
+++ b/src/framework/Area.cpp
@@ -12,6 +12,7 @@
 #include "Area.h"
 
 #include <iomanip>
+#include <limits>
 #include <sstream>
 
 namespace geo {

